lib/components/RunFrame/RunFrame.tsx

```diff
@@ -258,6 +270,19 @@ export const RunFrame = (props: RunFrameProps) => {
       props.onRenderFinished?.({ circuitJson })
       setAutoroutingGraphics({})
 
+      circuitJson
+        .filter((x: any) => x.type.includes("error") && x.error_type)
+        .forEach(
+          (x) =>
+            setErrors((prevErrors) => [
+              ...prevErrors,
+              {
+                error: (x as any).message || "Unknown error",
+                errorType: (x as any).error_type,
```

Comment:
Dont create new types that are similar to other types, in this case youre renaming errorType and losing error information. Just append the errors and make sure your error type is structurally the same